src/GridCell/GridCell.js

```diff
@@ -37,13 +89,13 @@ export const GridCell = ({ className, children, ref, ...props }) => {
   }
 
   // props.rowSpan
-  if (props.rowSpan && gridRowSpan.includes(props.rowSpan)) {
-    levers.rowSpan = `su-row-span-${props.rowSpan}`;
-    if (props.rowSpan === 'auto') {
-      levers.rowSpan = `su-row-auto`;
+  if (props.row && gridRowSpan.includes(props.row)) {
+    levers.row = `su-row-span-${props.row}`;
+    if (props.row === 'auto') {
+      levers.row = `su-row-auto`;
     }
     else {
-      levers.rowSpan = `su-row-span-${props.rowSpan}`;
+      levers.row = `su-row-span-${props.row}`;
     }
   }
```

Comment:
No breakpoints for row?

Reply:
I feel like we have lots of props already, and rowSpan is used rarely so I'm thinking that if people really need to use responsive rowSpan they could add it to the classname 🤔

Reply:
Ok, leave a code comment to that extent. I can imagine someone a couple years down the line wondering how to do this for their edge case.